server/src/main/kotlin/com/trib3/server/filters/RequestIdFilter.kt

```diff
@@ -77,11 +75,15 @@ class RequestIdFilter : Filter {
     override fun destroy() = Unit
 
     override fun doFilter(request: ServletRequest, response: ServletResponse, chain: FilterChain) {
-        withRequestId {
-            when (response) {
-                is HttpServletResponse -> response.setHeader(REQUEST_ID_HEADER, getRequestId())
-                else -> log.warn("Couldn't set request id header for {}", response::class.java)
+        val clientUUID = try {
+            (request as? HttpServletRequest)?.getHeader(REQUEST_ID_HEADER)?.let {
+                UUID.fromString(it).toString()
             }
+        } catch (e: IllegalArgumentException) {
```

Comment:
what case is this? maybe log? why remove the other log.warn?

Reply:
This case is a non-uuid requestId being sent in the headers.  mostly cause we want them to be unique, so forcing a uuid format should get us most of the way there.  I'll add a log for that.  the other log warn will never happen but was an artifact of the HttpServletRequest vs/ ServletRequest interface.